controllers/metallb_controller.go

```diff
@@ -80,11 +80,10 @@ func (r *MetallbReconciler) SetupWithManager(mgr ctrl.Manager) error {
 func (r *MetallbReconciler) syncMetalLBResources(config *metallbv1alpha1.Metallb) error {
 	logger := r.Log.WithName("syncMetalLBResources")
 	logger.Info("Start")
-	// var err error
-	objs := []*uns.Unstructured{}
```

Comment:
why objs was set twice? (line 84 and line 88) 
is removing it related to this commit?

Reply:
so maybe removing it should be in a separate commit?

Reply:
I would generally agree, but since it's a simple and easy fix and we are in the early stage of the deployment, I think it's fine doing it here